packages/core/src/tools/error.spec.ts

```diff
@@ -71,3 +71,24 @@ describe('formatUnknownError', () => {
     expect(formatted.message).toEqual('Uncaught {"foo":"bar"}')
   })
 })
+
+describe('createHandlingStackTrace', () => {
+  let stackTrace: string
+  function internalCall() {
+    stackTrace = createHandlingStackTrace()
```

Comment:
```suggestion
    handlingStack = createHandlingStack()
```

it could be nice to always refer to it as "handling stack" and remove all the "trace" references